### PR TITLE
fibonacci_series update

### PR DESCRIPTION
**Pull request by Adambear1**

### CODE REVIEW

**Comment by Adambear1** [comment]
Mailrooms with tests

File: students/ABirgenheier/lesson02/fibonacci_series.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    assert sum_series(2, 3, 2) == 5
+    assert sum_series(3, 3, 2) == 7
+    assert sum_series(4, 3, 2) == 12
+    assert sum_series(5, 3, 2) == 19
+
+    print("tests passed")
+
+
+if __name__ == "__main__":
+    run_assert()
```
Comment (charlesrobison) [inline students/ABirgenheier/lesson02/fibonacci_series.py:85]: Great!

File: students/ABirgenheier/lesson04/mailroom_2.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
+import sys
+# Donor dict
+donar_list = [
+    ("Mike", [200, 150, 50]),
+    ("Tony", [150, 50, 250]),
+    ("Sarah", [150, 150, 150]),
+]
```
Comment (charlesrobison) [inline students/ABirgenheier/lesson04/mailroom_2.py:7]: This should be a dictionary, starting from this version forward

File: students/ABirgenheier/lesson04/mailroom_2.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        prompt = input(
+            "What would you like to do?:\n"
+            "s - Send a thank you\n"
+            "c - Create a report\n"
+            "sa - Send a thank you to all donors\n"
+            "q - Quit\n")
+        if prompt.lower() in ["s", "c", "sa", "q"]:
+            menu[prompt.lower()]()
+        else:
+            print("\nPlease enter s, c, sa or q\n")
```
Comment (charlesrobison) [inline students/ABirgenheier/lesson04/mailroom_2.py:85]: Getting errors when trying to run this code - is it working for you?  Unable to run report, add new donor amounts, etc.  Please take another look and let me know if you have questions.

File: students/ABirgenheier/lesson05/mailroom_3.py [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+    while True:
+        response = input(main_prompt)
+        try:
+            switch_func_dict.get(response.lower())()
+        except (ValueError, TypeError):
+            print('\n--- -> Invalid Selection: Please select a correct input option.')
+
+
+if __name__ == '__main__':
+    main()
```
Comment (charlesrobison) [inline students/ABirgenheier/lesson05/mailroom_3.py:102]: This version works great!

File: students/ABirgenheier/lesson06/test_mailroom_4.py [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    assert os.path.exists(file2)
+
+
+if __name__ == "__main__":
+    test_donars()
+    test_list_of_donars()
+    test_donations()
+    test_thank_you_message()
+    test_send_letters_to_all()
+    print("All tests were successfully passed. Congratulations!!!")
```
Comment (charlesrobison) [inline students/ABirgenheier/lesson06/test_mailroom_4.py:52]: Not all tests pass here - please take another look.

**Review by charlesrobison — APPROVED** [review]
Code looks really good but having some issues with mailroom.  Please review and let me know if you have questions.  Also, make sure you submit your lesson 4 mailroom on the course site for grading.

Ready to merge

**Pull request merged**